src/clang_parser.cpp

```diff
@@ -321,6 +321,11 @@ bool ClangParser::parse(ast::Program *program, BPFtrace &bpftrace, std::vector<s
       .Contents = stdint_h,
       .Length = stdint_h_len,
     },
+    {
+      .Filename = ASM_GOTO_WORKAROUND_PATH,
```

Comment:
```suggestion
      .Filename = "/bpftrace/include/" ASM_GOTO_WORKAROUND_H,
```
To try and clarify what I was talking about in the other comment